assets/js/blocks/featured-items/featured-category/editor.scss

```diff
@@ -4,3 +4,7 @@
 	@extend %with-media-controls;
 	@extend %with-resizable-box;
 }
+
+.editor-styles-wrapper .wp-block-woocommerce-featured-category {
+	@extend %wp-block-featured-item;
+}
```

Comment:
If the purpose of this ruleset is only intended to fix #6489, I wonder if we should make it explicit in some way (maybe with a comment?).

Also, not blocking but wondering if instead of implementing this work-around here, we could update Storefront to avoid causing this issue. The problem seems to be that [this ruleset](https://github.com/woocommerce/storefront/blob/ed2530f09d447c8f94e39569a9bb3fbd38638d49/inc/customizer/class-storefront-customizer.php#L1070-L1072) has too much specificity and messes up the Featured Product/Category blocks, right? Maybe we could refactor that code from Storefront in some way? (I didn't really investigate that, just thinking out loud in case you have an idea, @danielwrobert :slightly_smiling_face: )

Reply:
> If the purpose of this ruleset is only intended to fix https://github.com/woocommerce/woocommerce-blocks/issues/6489, I wonder if we should make it explicit in some way (maybe with a comment?).

@Aljullu this still really doesn't fix #6489. I really only added this ruleset in to add a little bit more specificity to the styles in the editor, more generally, but it should still work the same way without this.

Thinking about it more, I'll go ahead and remove this ruleset from the editor stylesheet because it's not really consistent with everything else and this styling is intended for both the frontend and admin.

> Also, not blocking but wondering if instead of implementing this work-around here, we could update Storefront to avoid causing this issue.

Yes, that's 100% my thinking. IMO, the theme styles should be responsible to ensure they are not unintentionally overriding global (or other) settings. I noted this on the #7042 but should have reiterated my point here.

**So next steps:** I'll go ahead and remove the editor styles from this PR, as they're not really necessary, and merge. Then I'll make a note on #6489 (for posterity) and open a new issue on Storefront to address the problem you noted with the [overly specific ruleset](https://github.com/woocommerce/storefront/blob/ed2530f09d447c8f94e39569a9bb3fbd38638d49/inc/customizer/class-storefront-customizer.php#L1070-L1072) there.

Reply:
I'm a bit confused, this PR seems to reintroduce #6489. It's not clear to me if that's a known issue or not.

![imatge](https://user-images.githubusercontent.com/3616980/188663358-3a60e724-1c28-4e54-9a78-d3521107958e.png)

> **So next steps:** I'll go ahead and remove the editor styles from this PR, as they're not really necessary, and merge. Then I'll make a note on #6489 (for posterity) and open a new issue on Storefront to address the problem you noted with the [overly specific ruleset](https://github.com/woocommerce/storefront/blob/ed2530f09d447c8f94e39569a9bb3fbd38638d49/inc/customizer/class-storefront-customizer.php#L1070-L1072) there.

It sounds good to me, but wondering if we should prioritize this fix in Storefront. Otherwise that would be a regression. :confused:

Reply:
> I'm a bit confused, this PR seems to reintroduce https://github.com/woocommerce/woocommerce-blocks/issues/6489. It's not clear to me if that's a known issue or not.

@Aljullu that's correct. The issue is that the way the initial fix for #6489 was handled (via inline style attributes) prevents the global styles from being applied in the editor - which can be more confusing for users, IMO.

The idea here is to revert back to the original color rules and then handle the issue in #6489 on the Storefront side. Generally, it seems better to handle theme-specific issues from the theme side, since we won't be able to account for what different themes do that may override our plugin styles.

I agree that this should be prioritized in Storefront. I am planning on getting that going today so it doesn't get put on the back-burner and the regression gets rolled out to our users.

I can even hold off on merging this until the Storefront PR is further along, if you think that'd be better?

Additionally, @sunyatasattva [suggested a broader conversation on P2](https://github.com/woocommerce/woocommerce-blocks/issues/7042#issuecomment-1237917157) around how we prioritize styles overall, which I plan to post as a follow-up as well.

Reply:
Your explanation makes sense, sorry if I misunderstood it before. Your next steps look good to me.

> I can even hold off on merging this until the Storefront PR is further along, if you think that'd be better?

I will leave this up to you. Storefront has a shorter release process, so I think we are good if we merge this PR as-is and work on a fix in SF afterwards.

Reply:
Sounds good, thanks for the follow-up! I've created [an Issue on Storefront](https://github.com/woocommerce/storefront/issues/2036) and assigned myself to address ASAP.

Reply:
Yea just to chime in here as well, I think it is unclear what is the expected hierarchy between styles, and who should be concerned of being compatibile with what. Should themes make sure they are compatible with plugins or viceversa? I think we should probably come up with a scalable solution for these cases in general. 🤔

Reply:
I'm 100% with you: definitely default color preventing Global Styles is a mistake. Not providing a sensible default… I'd be against that. How all these variables interact with each other… that's where it gets really tricky and it's definitely a philosophical question at this point :)

Reply:
> I'm 100% with you: definitely default color preventing Global Styles is a mistake.

Since this seems to be the general consensus, I will go ahead and merge this PR to resolve that problem.

We can then continue to explore the best route forward for the text overriding issue with Storefront in [this discussion](https://github.com/woocommerce/storefront/pull/2039).